src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -573,9 +578,9 @@ public static HttpMethod ToHttpMethod(this OperationType key)
             };
         }
 
-        public static FormulaType GetReturnType(this OpenApiOperation openApiOperation)
+        public static FormulaType GetReturnType(this OpenApiOperation openApiOperation, ConnectorCompatibility compatibility)
```

Comment:
>public static FormulaType GetReturnType(this OpenApiOperation openApiOperation, ConnectorCompatibility compatibility) [](http://example.com/codeflow?start=8&length=117)

do we need to worry abt this breaking change? I'll check if Plugin uses this...

Reply:
nor cards

Reply:
Thanks for checking guys